docs/android/location-based-notifications.md

```diff
@@ -7,10 +7,12 @@ Add those permissions in your `AndroidManifest.xml`:
 	android:maxSdkVersion="22" />
 <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 ```
-And add the following NuGet package:
-
+And add the two following NuGet packages:
+```
 - Xamarin.GooglePlayServices.Location (min version: 57.1104.0-beta1)
 
+- Xamarin.Firebase.Messaging (min version: 57.1104.0-beta1)
```

Comment:
Since this was creating build errors, not related to the location features, move this package to the setup.md